### PR TITLE
Integrated padawan.php

### PR DESCRIPTION
**Pull request by mkusher**
#46

- [x] Added index generation command
- [x] Added plugin add and remove commands
- [x] Added server start, stop and restart commands
- [x] Added PadawanCompleter listener
- [x] Added padawan python client
- [x] Update README.md
- [x] Add configurable options

### CODE REVIEW

File: PHP Companion.py [hunk 1]
```diff
@@ -5,3 +5,8 @@
 from .php_companion.commands.replace_fqcn_command import ReplaceFqcnCommand
 from .php_companion.commands.goto_definition_scope import GotoDefinitionScopeCommand
 from .php_companion.commands.insert_constructor_prop import InsertConstructorPropertyCommand
+from .php_companion.commands.padawan_index_generate_command import PadawanGenerateIndexCommand
```
Comment (erichard) [inline PHP Companion.py]: Wrong filename here, it's padawan_generate_index_command.py

Comment (mkusher) [inline PHP Companion.py]: updated

**Comment by mkusher** [comment]
@erichard is there anything you want to add to this PR?

**Comment by erichard** [comment]
Thanks !

**Pull request merged**